PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -1949,7 +1949,7 @@ public void AddShapeButtonClick(Office.IRibbonControl control)
             customShape.AddCustomShape(shapeName, shapeFullName, true);
         }
 
-        private static CustomShapePane InitCustomShapePane()
+        private static CustomShapePane InitCustomShapePane(bool toggleVisible = false)

```

Comment:
Yeah, this function is also being called by `AddShapeButtonClick` which is when the user tries to add a shape to shapes lab. We don't want the pane to closed when the user adds shapes, so I made a boolean to check.